backend/src/main/java/de/neuefische/app/playlist/PlaylistDTO.java

```diff
@@ -0,0 +1,22 @@
+package de.neuefische.app.playlist;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+public class PlaylistDTO {
+
+    private String name;
+    private String spotifyId;
+    private List<PlaylistTrack> tracks;
```

Comment:
PlaylistTrack and PlaylistImage are the classes that you also use in the PlaylistData class. If you seperate data and DTO you should do it all the way.

Reply:
Ok, I am going to create a PlaylistTrackDTO and ImageDTO